certificates/create_ca.go

```diff
@@ -81,9 +109,16 @@ func generateCACertificate(keysize int, years int, days int, outputDir string) e
 		return fmt.Errorf("could not generate RSA private key: %s", err.Error())
 	}
 
-	keyID := generateKeyIDFromRSAPublicKey(privateKey.N, privateKey.E)
-
 	cn := fmt.Sprintf("EventStoreDB CA %s", serialNumber.Text(16))
+	subjectKeyID := generateKeyIDFromRSAPublicKey(privateKey.N, privateKey.E)
+	authorityKeyID := subjectKeyID
+	maxPathLen := 1
```

Comment:
@pvanbuijtene just confirming - there was a bug in the previous code - the value of maxPathLen should have been 0 instead of 1 to prevent any intermediate CAs from being present since the path length includes only CA certs and excludes the leaf certificate. (RFC: https://www.rfc-editor.org/rfc/rfc5280#section-4.2.1.9)

Reply:
A value of 1 seems ok here - unless we want to support testing scenarios where more than one intermediate certificate is present? 2 sounds like a reasonable value to cover 99.9% of cases

Reply:
@shaan1337 sounds good, updated the code.